subxt/src/blocks/mod.rs

```diff
@@ -14,3 +14,6 @@ pub use crate::backend::BlockRef;
 pub use block_types::Block;
 pub use blocks_client::BlocksClient;
 pub use extrinsic_types::{ExtrinsicDetails, ExtrinsicEvents, Extrinsics, StaticExtrinsic};
+
+// We get account nonce info in tx_client, too, so re-use the logic:
+pub(crate) use block_types::get_account_nonce;
```

Comment:
```suggestion
pub use block_types::get_account_nonce;
```
I would like to use this function from my lib please :pray:

Reply:
Oh but you can via `client.blocks().at_latest().account_nonce()` or `client.blocks().at(block_hash).account_nonce()` (there is one downside currently which is that it downloads the block header though, so yeah I'm not completely sold on this either; I forgot about that)